plan-patches/cf-lite-azure/README.md

```diff
@@ -0,0 +1,72 @@
+# Patch: cf-azure
+
+Steps to deploy cf-lite with bbl:
+
+1. Follow the normal steps to bbl up with a patch.
+
+    ```bash
+    export BOSH_BOOTLOADER=<YOUR BOSH BOOTLOAER PATH>
```

Comment:
Could you please correct this minor typo? BOOTLOAER -> BOOTLOADER

Reply:
changed, thanks :)